verbose_requirements.txt

```diff
@@ -67,6 +67,7 @@ statsmodels==0.14.2
 tensorboard==2.16.2
 tensorboard-data-server==0.7.2
 tensorflow==2.16.2
+torch==2.3.1
```

Comment:
I suggest the following:

add the pytorch to the setup.py file.
check if everything works fine ( with an `pip install .` on an empty virtual environment and without `pip install -r verbose_requirements`) 
if it does, do a pip freeze, so we have a record of what versions worked fine, should we have a problem at a later date.

Reply:
Great, it does work! I saved the ``pip freeze`` in ``verbose_requirements_torch.txt``